lib/ecto/query/builder.ex

```diff
@@ -544,6 +555,21 @@ defmodule Ecto.Query.Builder do
   end
 
   @doc """
+  Called by escaper at runtime to verify binaries.
+  """
+  def binary!(v) do
```

Comment:
We should probably call it "unsafe_fragment!" since we are mentioning the fragment in the error message.